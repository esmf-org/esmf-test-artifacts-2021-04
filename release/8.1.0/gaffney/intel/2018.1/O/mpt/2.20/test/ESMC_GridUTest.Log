20210306 094757.090 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210306 094757.090 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210306 094757.090 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210306 094757.090 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210306 094757.090 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_47-22-gffbbe59
20210306 094757.090 INFO             PET0 ESMF library build date/time: "Mar  6 2021" "09:27:29"
20210306 094757.090 INFO             PET0 ESMF library build location : /p/work1/mpotts/intel_2018.1_mpt_O
20210306 094757.090 INFO             PET0 ESMF_COMM                   : mpt
20210306 094757.106 INFO             PET0 ESMF_MOAB                   : enabled
20210306 094757.106 INFO             PET0 ESMF_LAPACK                 : enabled
20210306 094757.106 INFO             PET0 ESMF_NETCDF                 : enabled
20210306 094757.106 INFO             PET0 ESMF_PNETCDF                : disabled
20210306 094757.106 INFO             PET0 ESMF_PIO                    : enabled
20210306 094757.106 INFO             PET0 ESMF_YAMLCPP                : enabled
20210306 094757.106 INFO             PET0 Beginning Test, file ESMC_GridUTest.C, line 54
20210306 094757.106 INFO             PET0 NUMBER_OF_PROCESSORS 4
20210306 094757.138 INFO             PET0 PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
20210306 094757.236 INFO             PET0 PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
20210306 094757.236 INFO             PET0 PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
20210306 094757.236 INFO             PET0 PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
20210306 094757.236 INFO             PET0 PASS GridCreate, ESMC_GridUTest.C, line 206
20210306 094757.236 INFO             PET0 PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
20210306 094757.236 INFO             PET0 PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
20210306 094757.091 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.091 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210306 094757.091 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210306 094757.091 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210306 094757.091 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210306 094757.091 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.091 INFO             PET1 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_47-22-gffbbe59
20210306 094757.091 INFO             PET1 ESMF library build date/time: "Mar  6 2021" "09:27:29"
20210306 094757.091 INFO             PET1 ESMF library build location : /p/work1/mpotts/intel_2018.1_mpt_O
20210306 094757.091 INFO             PET1 ESMF_COMM                   : mpt
20210306 094757.092 INFO             PET1 ESMF_MOAB                   : enabled
20210306 094757.092 INFO             PET1 ESMF_LAPACK                 : enabled
20210306 094757.092 INFO             PET1 ESMF_NETCDF                 : enabled
20210306 094757.092 INFO             PET1 ESMF_PNETCDF                : disabled
20210306 094757.092 INFO             PET1 ESMF_PIO                    : enabled
20210306 094757.092 INFO             PET1 ESMF_YAMLCPP                : enabled
20210306 094757.092 INFO             PET1 Beginning Test, file ESMC_GridUTest.C, line 54
20210306 094757.092 INFO             PET1 NUMBER_OF_PROCESSORS 4
20210306 094757.138 INFO             PET1 PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
20210306 094757.232 INFO             PET1 PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
20210306 094757.232 INFO             PET1 PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
20210306 094757.232 INFO             PET1 PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
20210306 094757.232 INFO             PET1 PASS GridCreate, ESMC_GridUTest.C, line 206
20210306 094757.232 INFO             PET1 PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
20210306 094757.232 INFO             PET1 PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
20210306 094757.090 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210306 094757.090 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210306 094757.090 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210306 094757.090 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210306 094757.090 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET2 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_47-22-gffbbe59
20210306 094757.090 INFO             PET2 ESMF library build date/time: "Mar  6 2021" "09:27:29"
20210306 094757.090 INFO             PET2 ESMF library build location : /p/work1/mpotts/intel_2018.1_mpt_O
20210306 094757.090 INFO             PET2 ESMF_COMM                   : mpt
20210306 094757.092 INFO             PET2 ESMF_MOAB                   : enabled
20210306 094757.092 INFO             PET2 ESMF_LAPACK                 : enabled
20210306 094757.092 INFO             PET2 ESMF_NETCDF                 : enabled
20210306 094757.092 INFO             PET2 ESMF_PNETCDF                : disabled
20210306 094757.092 INFO             PET2 ESMF_PIO                    : enabled
20210306 094757.092 INFO             PET2 ESMF_YAMLCPP                : enabled
20210306 094757.092 INFO             PET2 Beginning Test, file ESMC_GridUTest.C, line 54
20210306 094757.092 INFO             PET2 NUMBER_OF_PROCESSORS 4
20210306 094757.138 INFO             PET2 PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
20210306 094757.234 INFO             PET2 PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
20210306 094757.234 INFO             PET2 PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
20210306 094757.234 INFO             PET2 PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
20210306 094757.235 INFO             PET2 PASS GridCreate, ESMC_GridUTest.C, line 206
20210306 094757.235 INFO             PET2 PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
20210306 094757.235 INFO             PET2 PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
20210306 094757.090 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210306 094757.090 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210306 094757.090 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210306 094757.090 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210306 094757.090 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210306 094757.090 INFO             PET3 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_47-22-gffbbe59
20210306 094757.090 INFO             PET3 ESMF library build date/time: "Mar  6 2021" "09:27:29"
20210306 094757.090 INFO             PET3 ESMF library build location : /p/work1/mpotts/intel_2018.1_mpt_O
20210306 094757.091 INFO             PET3 ESMF_COMM                   : mpt
20210306 094757.091 INFO             PET3 ESMF_MOAB                   : enabled
20210306 094757.091 INFO             PET3 ESMF_LAPACK                 : enabled
20210306 094757.091 INFO             PET3 ESMF_NETCDF                 : enabled
20210306 094757.091 INFO             PET3 ESMF_PNETCDF                : disabled
20210306 094757.091 INFO             PET3 ESMF_PIO                    : enabled
20210306 094757.091 INFO             PET3 ESMF_YAMLCPP                : enabled
20210306 094757.091 INFO             PET3 Beginning Test, file ESMC_GridUTest.C, line 54
20210306 094757.091 INFO             PET3 NUMBER_OF_PROCESSORS 4
20210306 094757.138 INFO             PET3 PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
20210306 094757.230 INFO             PET3 PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
20210306 094757.231 INFO             PET3 PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
20210306 094757.231 INFO             PET3 PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
20210306 094757.231 INFO             PET3 PASS GridCreate, ESMC_GridUTest.C, line 206
20210306 094757.231 INFO             PET3 PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
20210306 094757.231 INFO             PET3 PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
