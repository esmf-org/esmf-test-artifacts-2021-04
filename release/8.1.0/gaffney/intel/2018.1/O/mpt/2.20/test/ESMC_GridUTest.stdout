Beginning Test, file ESMC_GridUTest.C, line 54
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_GridUTest.C, line 54
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_GridUTest.C, line 54
Beginning Test, file ESMC_GridUTest.C, line 54
NUMBER_OF_PROCESSORS 4
NUMBER_OF_PROCESSORS 4
PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
PASS GridCreateCubedSphere, ESMC_GridUTest.C, line 139
PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
PASS GridCreate, ESMC_GridUTest.C, line 206
PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
PASS GridCreate, ESMC_GridUTest.C, line 206
PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
PASS GridCreate, ESMC_GridUTest.C, line 206
PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
PASS GridWrite(cubedsphere), ESMC_GridUTest.C, line 150
PASS GridCreateGetCoord(cubedsphere), ESMC_GridUTest.C, line 159
PASS GridDestroy(cubedsphere), ESMC_GridUTest.C, line 185
PASS GridCreate, ESMC_GridUTest.C, line 206
PASS GridCreate_periodicDim_1, ESMC_GridUTest.C, line 221
PASS GridCreate_periodicDim_2, ESMC_GridUTest.C, line 243
MPT ERROR: Rank 0(g:0) received signal SIGSEGV(11).
	Process ID: 383381, Host: r2i5n20, Program: /p/work1/mpotts/intel_2018.1_mpt_O/test/testO/Linux.intel.64.mpt.default/ESMC_GridUTest
	MPT Version: HPE MPT 2.20  08/30/19 04:33:45

MPT: --------stack traceback-------
MPT: Attaching to program: /proc/383381/exe, process 383381
MPT: [Thread debugging using libthread_db enabled]
MPT: Using host libthread_db library "/lib64/libthread_db.so.1".
MPT: done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: 0x00002aaaad48018c in waitpid () from /lib64/libpthread.so.0
MPT: Missing separate debuginfos, use: debuginfo-install cyrus-sasl-lib-2.1.26-23.el7.x86_64 glibc-2.17-317.el7.x86_64 keyutils-libs-1.5.8-3.el7.x86_64 krb5-libs-1.15.1-50.el7.x86_64 libbitmask-2.0-sgi720r52.rhel76.x86_64 libcom_err-1.42.9-19.el7.x86_64 libcpuset-1.0-sgi720r102.rhel76.x86_64 libcurl-7.29.0-59.el7_9.1.x86_64 libgcc-4.8.5-44.el7.x86_64 libidn-1.28-4.el7.x86_64 libjpeg-turbo-1.2.90-8.el7.x86_64 libselinux-2.5-15.el7.x86_64 libssh2-1.8.0-4.el7.x86_64 libstdc++-4.8.5-44.el7.x86_64 nspr-4.25.0-2.el7_9.x86_64 nss-3.53.1-3.el7_9.x86_64 nss-softokn-freebl-3.53.1-6.el7_9.x86_64 nss-util-3.53.1-1.el7_9.x86_64 openldap-2.4.44-22.el7.x86_64 openssl-libs-1.0.2k-21.el7_9.x86_64 pcre-8.32-17.el7.x86_64 zlib-1.2.7-18.el7.x86_64
MPT: (gdb) #0  0x00002aaaad48018c in waitpid () from /lib64/libpthread.so.0
MPT: #1  0x00002aaab1960806 in mpi_sgi_system (
MPT: #2  MPI_SGI_stacktraceback (
MPT:     header=header@entry=0x7fffffff8040 "MPT ERROR: Rank 0(g:0) received signal SIGSEGV(11).\n\tProcess ID: 383381, Host: r2i5n20, Program: /p/work1/mpotts/intel_2018.1_mpt_O/test/testO/Linux.intel.64.mpt.default/ESMC_GridUTest\n\tMPT Version: H"...) at sig.c:340
MPT: #3  0x00002aaab1960a02 in first_arriver_handler (signo=signo@entry=11, 
MPT:     stack_trace_sem=stack_trace_sem@entry=0x2aaab9320080) at sig.c:489
MPT: #4  0x00002aaab1960d9b in slave_sig_handler (signo=11, 
MPT:     siginfo=<optimized out>, extra=<optimized out>) at sig.c:565
MPT: #5  <signal handler called>
MPT: #6  0x00002aaab18cec9e in MPI_SGI_comm_fetch (comm=comm@entry=1140850689)
MPT:     at comm.c:41
MPT: #7  MPI_SGI_comm_get (comm=comm@entry=1140850689)
MPT:     at ../../../../include/comm.h:182
MPT: #8  MPI_SGI_comm_hbt (comm=comm@entry=1140850689) at comm.c:311
MPT: #9  0x00002aaab18d58c8 in MPI_SGI_comm_set_attr (attr_value=0x0, key=14, 
MPT:     comm=1140850689) at comm_set_attr.c:16
MPT: #10 PMPI_Comm_set_attr (comm=1140850689, key=14, attr_value=0x0)
MPT:     at comm_set_attr.c:68
MPT: #11 0x00002aaab28f22f0 in H5_init_library ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/hdf5-1.10.6/lib/libhdf5.so.103
MPT: #12 0x00002aaab29a27a9 in H5Eset_auto2 ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/hdf5-1.10.6/lib/libhdf5.so.103
MPT: #13 0x00002aaab08a027c in nc4_hdf5_initialize ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/netcdf-4.7.4/lib/libnetcdf.so.18
MPT: #14 0x00002aaab08a97bc in NC_HDF5_initialize ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/netcdf-4.7.4/lib/libnetcdf.so.18
MPT: #15 0x00002aaab08191a8 in nc_initialize ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/netcdf-4.7.4/lib/libnetcdf.so.18
MPT: #16 0x00002aaab081caac in NC_open ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/netcdf-4.7.4/lib/libnetcdf.so.18
MPT: #17 0x00002aaab081c7d7 in nc_open ()
MPT:    from /app/CSE/CSEIntel.20200724/Datamodels/netcdf-4.7.4/lib/libnetcdf.so.18
MPT: #18 0x00002aaab0c53f5d in nf_open (path='data/T42_grid.nc', mode=0, ncid=0, 
MPT:     .tmp.PATH.len_V$30e=16) at nf_control.F90:189
MPT: #19 0x00002aaab0ca6871 in netcdf::nf90_open (path='data/T42_grid.nc', mode=0, 
MPT:     ncid=0, 
MPT:     chunksize=<error reading variable: Cannot access memory at address 0x0>, 
MPT:     cache_size=<error reading variable: Cannot access memory at address 0x0>, 
MPT:     cache_nelems=<error reading variable: Cannot access memory at address 0x0>, 
MPT:     cache_preemption=<error reading variable: Cannot access memory at address 0x0>, comm=<error reading variable: Cannot access memory at address 0x0>, 
MPT:     info=<error reading variable: Cannot access memory at address 0x0>, 
MPT:     .tmp.PATH.len_V$1016=16) at netcdf4_file.f90:64
MPT: #20 0x00002aaaac0ef254 in esmf_ioscripmod_mp_esmf_scripinq_ ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #21 0x00002aaaabfee253 in esmf_gridmod_mp_esmf_gridcreatefrmscrip_ ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #22 0x00002aaaabff90ab in esmf_gridmod_mp_esmf_gridcreatefrmncfile_ ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #23 0x00002aaaabfa55a0 in f_esmf_gridcreatefromfile_ ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #24 0x00002aaaab51bb0a in ESMCI::Grid::createfromfile(char const*, int, int*, int*, int*, ESMC_InterArrayInt*, int*, int*, ESMC_IndexFlag*, int*, char const*, char const**, int*) ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #25 0x00002aaaab147fc2 in ESMC_GridCreateFromFile ()
MPT:    from /p/work1/mpotts/intel_2018.1_mpt_O/lib/libO/Linux.intel.64.mpt.default/libesmf.so
MPT: #26 0x0000000000406b55 in main ()
MPT: (gdb) A debugging session is active.
MPT: 
MPT: 	Inferior 1 [process 383381] will be detached.
MPT: 
MPT: Quit anyway? (y or n) [answered Y; input not from terminal]
MPT: Detaching from program: /proc/383381/exe, process 383381
MPT: [Inferior 1 (process 383381) detached]

MPT: -----stack traceback ends-----
MPT: On host r2i5n20, Program /p/work1/mpotts/intel_2018.1_mpt_O/test/testO/Linux.intel.64.mpt.default/ESMC_GridUTest, Rank 0, Process 383381: Dumping core on signal SIGSEGV(11) into directory /p/work1/mpotts/intel_2018.1_mpt_O/test/testO/Linux.intel.64.mpt.default
MPT ERROR: MPI_COMM_WORLD rank 0 has terminated without calling MPI_Finalize()
	aborting job
MPT: Received signal 11
